<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEngineDialog</class>
 <widget class="QDialog" name="AddEngineDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Engine</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelEngineName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>110</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Engine name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="engineName">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>10</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelEngineProtocol">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>110</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Engine protocol</string>
   </property>
  </widget>
  <widget class="QComboBox" name="engineProtocol">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>XBoard</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UCI</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UCCI</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Qianhong</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>USI</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelEnginePath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>110</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Engine path</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="enginePath">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>366</x>
     <y>130</y>
     <width>27</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelGameVariant">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Game variant</string>
   </property>
  </widget>
  <widget class="QComboBox" name="gameVariant">
   <property name="geometry">
    <rect>
     <x>278</x>
     <y>50</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Chess</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Xiangqi</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Shogi</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sho Shogi</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chu Shogi</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mini Shogi</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelEngineParameters">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Engine parameters</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="engineParameters">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>engineName</tabstop>
  <tabstop>gameVariant</tabstop>
  <tabstop>engineProtocol</tabstop>
  <tabstop>enginePath</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>engineParameters</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddEngineDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddEngineDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
